Makefile

```diff
@@ -71,6 +71,13 @@ clean: ## Cleanup build artifacts and tool binaries.
 
 ##@ Build
 
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
```

Comment:
@acornett21 this commit is required to allow run make install.
without this fix it fails locally for me.